app/__init__.py

```diff
@@ -136,9 +137,25 @@ def create_app(flask_config_name=None, config_override={}, testing=False, **kwar
     # Replace app.config (flask.Config) with our HoustonFlaskConfig version
     configure_using_houston_flask_config(app)
 
+    # Set up celery
+    redis_uri = f'redis://{app.config["REDIS_HOST"]}/0'
+    app.celery = Celery('houston', broker=redis_uri, backend=redis_uri)
```

Comment:
Please help me understand this? Is this now listening on host/port of REDIS_HOST?
If so, how does this not clash with the tus code which is also using the same REDIS_HOST?

Reply:
Yeah it is currently using the same redis database as tus.  We can change the `/0` to `/1` to use a different database in the same redis instance.